src/main/java/com/gorlah/kappabot/integration/discord/DiscordRequestMetadata.java

```diff
@@ -8,10 +9,17 @@
 public class DiscordRequestMetadata implements BotRequestMetadata {
 
     private final String author;
+    private final String authorMention;
     private final String message;
 
     DiscordRequestMetadata(MessageReceivedEvent event) {
         this.author = event.getAuthor().getName();
+        this.authorMention = event.getAuthor().getAsMention();
         this.message = event.getMessage().getContentRaw();
     }
+
+    @Override
+    public BotRequestEndpoint getEndpoint() {
+        return BotRequestEndpoint.DISCORD;
```

Comment:
After seeing how this is used, I think you could make `BotRequestEndpoint` a regular `class` instead of an `enum`, and have instances of it declared as constants. That would give the flexibility for future plugins to provide their own.

Reply:
This is handled by changes in master, where the source is represented by a `String`.